public/css/main.css

```diff
@@ -32,6 +220,3 @@ h1 {
   flex-shrink: 0;
 }
 
-.pr-item {
-  word-break: break-word;
-}
```

Comment:
How come we're losing this?

Reply:
This was added as part of https://github.com/jenkoian/hacktoberfest-checker/pull/172 so think we should put this back.